File: bookstore-backend/src/main/java/com/example/bookstore_db/dto/JwtResponse.java
package com.example.bookstore_db.dto;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class JwtResponse {
    private String accessToken;
    private String tokenType = "Bearer";
    private Long id;
    private String username;
    public JwtResponse(String accessToken, Long id, String username) {
        this.accessToken = accessToken;
        this.id = id;
        this.username = username;
    }
}

File: BookStore/app/src/main/java/com/example/bookstore/data/dto/response/JwtResponse.kt
package com.example.bookstore.data.dto.response

import com.google.gson.annotations.SerializedName

data class JwtResponse(
    @SerializedName("accessToken") val token: String,  // backend trả "accessToken"
    val tokenType: String? = null,
    @SerializedName("id") val userId: Long? = null,    // backend trả "id" (không phải "userId")
    val username: String? = null
)
